src/v2/Components/AuctionResultPerformance.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import {
+  Text,
+  ThemeProviderV3,
+  IncreaseIcon,
+  Flex,
+  DecreaseIcon,
+  Spacer,
+} from "@artsy/palette"
+
+export const AuctionResultPerformance = ({
+  value,
+  align,
+}: {
+  value: string | null
+  align: "left" | "right"
+}) => {
+  if (value === null) {
+    return null
+  }
+
+  const sign = value[0] === "-" ? "down" : "up"
+  const color = sign === "up" ? "green100" : "red100"
+  const text = sign === "up" ? value : value.slice(1)
+  const Arrow = sign === "up" ? IncreaseIcon : DecreaseIcon
+
+  return (
+    <ThemeProviderV3>
```

Comment:
Just making sure, do we still need this?

Reply:
@damassi

Reply:
Nope! since we've migrated everything over it should automatically pick up the theme.